pkg/e2etest/e2e_test.go

```diff
@@ -432,7 +428,12 @@ func runGoQuery(t *testing.T, res interface{}, args []string) {
 	require.Nil(t, command.Execute())
 	require.Nil(t, wr.Close())
 	<-copyDone
+
 	os.Stdout = old // restore the inital STDOUT
+	require.Nil(t, logging.Init(logging.LevelWarn, logging.EncodingLogfmt,
```

Comment:
@els0r FYI: This re-initialization of the logger interestingly was required to avoid losing all subsequent logs after the `runGoQuery()` method has been called for the first time (before it was `INFO` level and would only show logs from `goProbe` once until this method was called). I don't know why, but apparently the `logging` package doesn't take lightly to me "stealing" `os.Stdout` and then resetting it (interestingly, all other output to `STDOUT` works afterwards as expected). My handling here is a bit crude and not the recommended way (but unfortunately I don't have any other option the way `goQuery` throws its messages), but I was wondering if in your opinion this warrants investigation?

Reply:
Interesting. I'd say let's park it for now. Unless we find out that this type of manipulation is more commonplace than we thought.